src/.nuget/Microsoft.Private.Tests.Native.CoreCLR/Microsoft.Private.Tests.Native.CoreCLR.pkgproj

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
With @echesakov's work we are now using SDK projects so we should avoid these legacy node attributes.

Reply:
The packaging projects are still non-SDK projects.